packages/core/src/configuration.ts

```diff
@@ -379,6 +379,7 @@ const pieChart: PieChartOptions = Tools.merge({}, chart, {
 			enabled: true,
 		},
 		alignment: Alignments.LEFT,
+		sortFunction: null,
```

Comment:
```suggestion
		sortFunction: (a: any, b: any) => a.index - b.index
```

Reply:
this doesn't work because data inside `pieLayout` only have `{group: "YEL48 Q6XK YEL48", value: 10000}` which is data that put into `const pieLayoutData = pieLayout(displayData);` and I think the sort below at line 86 doesn't sort anything because when the data change d3 pie will using the sort function from the layout. I think the best way is to use null for default sort.